src/pages/home/report/ReportActionCompose.js

```diff
@@ -437,8 +537,21 @@ class ReportActionCompose extends React.Component {
      * @param {Object} e
      */
     triggerHotkeyActions(e) {
-        // Do not trigger actions for mobileWeb or native clients that have the keyboard open because for those devices, we want the return key to insert newlines rather than submit the form
-        if (!e || this.props.isSmallScreenWidth || this.props.isKeyboardShown) {
+        if (e.key === 'Enter' && this.state.suggestedEmojis.length) {
+            this.suggestedEmojisSelect(this.state.highlightedEmojiIndex);
+            e.preventDefault();
+            return;
+        }
+        if (e.key === 'Escape' && this.state.suggestedEmojis.length) {
+            this.setState({
+                suggestedEmojis: [],
+                shouldShowSuggestionMenu: false,
+            });
+            e.preventDefault();
+            return;
+        }
+
+        if (!e) {
```

Comment:
This if statement will never be hit – we already tried to access `e.key` and `e.preventDefault` so if `e` is falsy we would have already thrown an error. I think this should either be moved so that it's the first line in the function or (better yet) removed altogether if it's not needed.